src/components/FeatureViews/Courses/CourseClasses.js

```diff
@@ -268,234 +273,223 @@ const CourseClasses = () => {
 
     return (
         <Grid item xs={12}>
-            <BackgroundPaper elevation={2}>
-                <Grid container justify="space-between" alignContent="center">
-                    <Grid item>
-                        <BackButton />
-                    </Grid>
-                    <Grid item>
-                        {accountType === 'PARENT'
-                            ? studentInCourse.map((student, i) => (
-                                  <StudentCourseLabel label={student} key={i} />
-                              ))
-                            : ''}
-                    </Grid>
+            <Grid container justify="space-between" alignContent="center">
+                <Grid item>
+                    <BackButton />
                 </Grid>
-                <Hidden xsDown>
-                    <hr />
-                </Hidden>
-                <Grid container>
-                    <Grid item xs={6}>
-                        <Typography
-                            align="left"
-                            className="heading"
-                            variant="h3"
-                            style={{ marginTop: '.65em' }}
-                        >
-                            {title}
-                        </Typography>
-                    </Grid>
-                    <Grid item xs={6}>
-                        <AccessControlComponent
-                            permittedAccountTypes={[
-                                USER_TYPES.admin,
-                                USER_TYPES.receptionist,
-                                USER_TYPES.instructor,
-                            ]}
-                        >
-                            <IconButton
-                                className={classes.editcoursebutton}
-                                size="small"
-                                component={Link}
-                                to={`/registration/form/course_details/${id}`}
-                            >
-                                <EditIcon />
-                            </IconButton>
-                        </AccessControlComponent>
-                    </Grid>
+                <Grid item>
+                    {accountType === 'PARENT'
+                        ? studentInCourse.map((student, i) => (
```

Comment:
change the ternary to an && since you're just rendering an empty string if condition is false